meetings/2022-06-16.md

```diff
@@ -43,17 +46,38 @@
 * SC: Left comments/issues/PRs based on this.
-
+* eP: If GET or HEAD includes that Link relation on every resource that responds with 2xx response that seems to solve the problem.
+* SC: May not be limited to 2xx.
 
 ### Discovery of Notification Subscription Metadata
 URL: https://github.com/solid/notifications/issues/58
 
 * SC: I'll summarise https://github.com/solid/notifications/issues/58#issuecomment-1151117524 .
-
+* CB: Each resource points to a notification discovery resource or if every resource in a container is managed by the same channel. If I subscribe to a container, I get a notification about the container. Is that intended to be the same or buggy?
+* CB: question was asked if every resource in the storage points to the same description resource with information about subscribing to notification channels
+* SC: Generally when the resource description changes. The assumption is that you get notification, so if the containement information changes you would get a notification.
+* SC: Forget about the prior/old/insecure websocket behaviour that's not the one described here.
+* eP: We may need to document use cases to subscribe to contained resources
+* RG: How much of this should be on the server and how much on the client? Have we thought about how to do this in terms of implementation efficiency and balancing that against with the ease of making a client.
+* SC: Not that I know of anything that's documented. Which is why we are hoping for implementation feedback to approach those questions.
+* ...: Myself I understood the protocol better after implementing some of it, and what things needs to be clarified in the spec or missing.
+* CB: Rahul has a point, the client side should always work, however if you imagine a web app that accesses a lot of resources on the server, it may affect scalability to have separate websockets connections.
+* eP: We should start with UC and track. With websocket connection, each resource requries subscription, the server could use source and just response - that's why we are optimising it. For the second one is an option for server to provide or not for aggregate subscription. It is mostly the container, as opposed to multiple topic.
+* SC: My understanding was that as soon as the description of the resource changes, it can trigger a notification.
+* eP: Chat room container, a new chat. But edited a chat line, I get a notification just by subscribing to a container. In a way a propogation.
+* SC: I think we do need to cicle back on that a bit with the Solid Protocol because there is this bit https://solidproject.org/TR/protocol#contained-resource-metadata . Lets come back to this.
+* SC: Right now Notification Protocol doesn't have a notion of Solid Storage. If we don't have that requirements where the channel descriptions are available from.
```

Comment:
```suggestion
* SC: I think we do need to cicle back on that a bit with the Solid Protocol because there is this bit: <https://solidproject.org/TR/protocol#contained-resource-metadata>. Lets come back to this.
* SC: Right now, Notification Protocol doesn't have a notion of Solid Storage. If we don't have that requirement, where are the channel descriptions available from?
```